app/views/submissions/view.html.erb

```diff
@@ -26,8 +26,6 @@
     <% end %>
 
     const editableUrl = "<%= url_for ([:quickSetScore, @course, @assessment]) %>"
-    const rootFiles = $('.file-list').children()
```

Comment:
With the new golden layout, these selectors are run before the elements exist. Remedied this by deferring evaluation until the values are needed.